airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1354,6 +1354,64 @@ def run_grant_dataset_view_access(self,
                 view_project, view_dataset, view_table, source_project, source_dataset)
             return source_dataset_resource
 
+    def insert_dataset(self, dataset_id="", project_id="", dataset_resource=None):
```

Comment:
Can you change its name (eg `create_empty_dataset`) to be in line with other methods such as `create_empty_table`

https://github.com/apache/incubator-airflow/blob/5be1fe9822ca9ed4111f9de8758cb9d6813d56e0/airflow/contrib/hooks/bigquery_hook.py#L227-L238